package.json

```
@@ -7,6 +7,7 @@
     "dev": "vite",
     "build": "vite build",
     "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "lint:fix": "npx prettier . --write",
```

Did you install `prettier`? This script will fail because this dependency is not installed.
```suggestion
    "lint:fix": "eslint --fix",
```